src/terms/terms.jl

```diff
@@ -60,8 +60,10 @@ breaks_symmetries(::Anyonic) = true
 
 # forces computes either nothing or an array forces[at][α] (by default no forces)
 compute_forces(::Term, ::AbstractBasis, ψ, occupation; kwargs...) = nothing
-# dynamical matrix for phonons computations (array dynmat[n_dim, n_atom, n_dim, n_atom])
+# dynamical matrix for phonons computations (array dynmat[3, n_atom, 3, n_atom])
 compute_dynmat(::Term, ::AbstractBasis, ψ, occupation; kwargs...) = nothing
+# variation of the Hamiltonian applied to orbitals for phonons computations
+compute_δHψ_sα(::Term, ::AbstractBasis, ψ, q, s, α; kwargs...) = nothing
```

Comment:
Better name, but needs some more details what all these things mean, what is the expected input / output etc. as this function is not as natural as the other ones (at least to me) ... see also my other comments)